### ISSUE TITLE
wireshark-dissector: Not identifying as TW with custom server

### ISSUE THREAD

**Issue by srdante** [heinrich5991/libtw2#67]
The dissector is actually not identifying packets as TW for a custom server implementation. Even using a default DDNet client, the sending packets are also not detected.

![image](https://user-images.githubusercontent.com/35383529/200370656-20ddda71-e90a-4985-913f-600403141400.png)

This is a simple python implementation that reproduces a "The server is full" response message:

```py

import socket

UDP_IP_ADDRESS = "127.0.0.1"
UDP_PORT_NO = 8304

serverSock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
serverSock.bind((UDP_IP_ADDRESS, UDP_PORT_NO))

while True:
    data, addr = serverSock.recvfrom(1024)
    serverSock.sendto(b"\x10\x00\x00\x04\x54\x68\x65\x20\x73\x65\x72\x76\x65\x72\x20\x69\x73\x20\x66\x75\x6c\x6c", addr)
```

**Comment by ChillerDragon** [comment]
Did you try running the server on port ``8303`` instead of ``8304``?

**Comment by heinrich5991** [comment]
That should not matter, I believe. I don't have a 8303 in the source code of the dissector.

**Comment by ChillerDragon** [comment]
Oh I missed 8cf29960091f303036fffd3ef98a185108716cd7

Back in the days I used to change that port if needed. But seems like its not needed any more nice!

**Comment by heinrich5991** [comment]
The problem is that it doesn't identify the TOKEN extension used by DDNet, I think.

**Comment by heinrich5991** [comment]
Btw, workaround is: right click on a Teeworlds packet → Decode as… → Current (none) → select "Teeworlds packet" / "Teeworlds 0.7 packet" as appropriate.

**Comment by heinrich5991** [comment]
Should be fixed by a337133d3eb022ac284b3d2e8fa93b9168836ef6.
